modules/control/warps.lua

```diff
@@ -37,9 +37,16 @@ Global.register(force_warps, function(tbl)
     force_warps = tbl
 end)
 
+-- Create a array of entity names that will be added to the remove filter
+local remove_warp_area_entity_names = {}
+for _, entity in pairs(config.entities) do
+    table.insert(remove_warp_area_entity_names, entity[1])
+end
+
 -- When a warp is updated change its chat tag and resort the warp order
```

Comment:
```suggestion
-- When a warp is updated change its chat tag and restore the warp order
```
I'm assuming resort is a typo here? Also yes I see this isn't technically in the diff but close enough :P